plugins/tours/tests/phpunit/integration/renderTourComments.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Tests for render_tour_comments().
+ *
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @package     spiralWebDb\CornerstoneTours\Tests\Integration
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Integration;
+
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+use function spiralWebDb\CornerstoneTours\render_tour_comments;
+
+/**
+ * @covers ::\spiralWebDb\CornerstoneTours\render_tour_comments
+ *
+ * @group   tours
+ */
+class Tests_RenderTourComments extends Test_Case {
+
+	/**
+	 * @dataProvider addTestData
+	 */
+	public function test_should_echo_meta_key_values_when_postmeta_exists( $post_data, $meta ) {
+		// Create and get the $tour_id using WordPress' factory method.
+		$tour_id = $this->factory->post->create( $post_data );
+
+		// Add post_meta to the database so we can call it.
+		add_post_meta( $tour_id, 'tour_comments', $meta['tour_comments']
+		);
+
+		$expected = (string) get_post_meta( $tour_id, 'tour_comments', true );
```

Comment:
This is not what's expected. Right? The expected value should be passed from the test data. Why? The actual expected value depends upon what is being tested from the test data's scenario. Pass it from the test data.

Huh?

Let's look at the code:

https://github.com/rgadon107/cornerstone/blob/b12d9c8aa0593a6a4a6d7af418e2a78c8234a1b3/plugins/tours/src/meta.php#L38-L46

What do you see? What are the scenarios?

1. If the post meta is empty, then the expected behavior is: the function bails out and returns void/null.
2. If there's post meta, then it returns it.
3. Notice the post meta is escaped through `esc_html()`. You want to test that it gets escaped too.

So there are 3 different scenarios to test. And you'll want to pass in different types of "tour comments" to validate it works as you expect.

Does that make sense?